### PR TITLE
Translated in Italian

### LINKED ISSUE #27: Can I translate this project in Italian?
I'm a native Italian, and I like this project. I would like contributing with you and I would like adding the translation in Italian, can I do it?

### PR DESCRIPTION
**Pull request by Sav22999**
Fixes #27

### CODE REVIEW

File: po/extra/it.po [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+msgid "Badges for your android app's README"
+msgstr "Distintivo per il README della tua app Android"
+
+#: data/com.github.rajkumaar23.badgie.appdata.xml.in:10
+msgid ""
+"Say NO to hardcoding! Generate badges for your android app's README with "
+"realtime data from Google Playstore such as Version, Number of Installs, "
+"Rating etc"
+msgstr ""
+"Dici NO all'hardcoding! Genera distintivi per il README della tua app "
```
Comment (rajkumaar23) [inline po/extra/it.po:56]:
```suggestion
"Dici NO alla hardcoding! Genera distintivi per il README della tua app "
```
This is what Google Translate suggests me. Please let me know if the suggestion is wrong.

**Review by rajkumaar23 — COMMENTED** [review]
Thanks for the contribution, @Sav22999 !

File: po/extra/it.po (hunk 1 above)
Comment (Sav22999) [reply po/extra/it.po:56]: The suggestion by Google is not wrong, but it's the "long form". We in Italian use, basically, just the first one.

It's like "I am" and "I'm" > Google will suggest always "I am", but "I'm" is correct as well

Comment (rajkumaar23) [reply po/extra/it.po:56]: Gotcha! Merging it 🎉

**Review by rajkumaar23 — APPROVED** [review]

**Pull request merged**